examples/pamessage1.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Instead of numbering the PA message examples, should we name them pamessage_video.json and so on?

Reply:
ok! Renamed files